README.md

```diff
@@ -1,6 +1,6 @@
 # Tau.jl
 
-<div align="center"><img src="https://rawgit.com/JuliaMath/Tau.jl/master/tau-2pi.svg" width="300"/></div><br/><br/>
+<div align="center"><img src="tau-2pi.svg" width="300"/></div><br/><br/>
```

Comment:
I was going to ask whether the URL change would allow the image to continue working outside GitHub, e.g. on https://juliapackages.com/p/tau or https://juliahub.com/ui/Packages/General/Tau, but it turns out the current one already doesn't work, so we might even end up fixing that; and even if we don't, things won't get worse than they already are, so this looks like a strict improvement. Thanks!

Reply:
I'm not sure reverting the change would fix the issue — the image was already broken with the previous URL. Do note, however, that it was a `rawgit.com` URL rather than a `raw.githubusercontent.com` one, so perhaps a change to the latter would indeed be beneficial. A PR to do that would be very welcome, but perhaps we should make a release first, to check if the simpler filename-only path works. WDYT, @giordano — would a patch release be OK for this?

Reply:
Yeah, it's probably fine.

Reply:
Would you mind making the release? :sweat_smile:  I'm a bit rusty on the process as it's been a while. (Or just point me towards the relevant docs. Do we create a tag for a commit in this repo and TagBot handles the PR to the registry, or is it the other way around?)

Reply:
1. Bump https://github.com/JuliaMath/Tau.jl/blob/e951380ac2c43b69b44af166e5b159e431b6a0b1/Project.toml#L3 to 2.0.1
2. in the page of the commit on the `master` branch which contains the bump, comment with
    ```
    @JuliaRegistrator register
    ```
3. sit back and relax

Reply:
After a small speed bump in #53 and #58, this has now been done via #57 — here's the commit: https://github.com/JuliaMath/Tau.jl/commit/a74455ebc7590c8dbbd2d0e0e01da2b96664d939

Reply:
@rwnobrega it seems like the simple file path works! Both the README and https://juliahub.com/ui/Packages/General/Tau show the image. (https://juliapackages.com/p/tau still shows the broken image, but its cache hasn't been updated yet, because in the page source code I can see the the attribute `data-canonical-src="https://rawgit.com/JuliaMath/Tau.jl/master/tau-2pi.svg"` in the `<img>` element, i.e. pointing to the URL prior to this PR and to the release of v2.0.1.) So I'd say we're good and there's no need for further changes! Thanks again for the contribution :)

Reply:
Great to hear that! Thank you as well. :smile: